### PR TITLE
Speed up weight loading by replacing istringstream with strtof

### PR DESCRIPTION
**Pull request by Akababa**
It loads the weights file 2-3x as fast on my machine.

### CODE REVIEW

**Comment by glinscott** [comment]
Thanks!  Can you validate that the `bench` output is the same before and after with the same network weights?

File: src/Network.cpp [hunk 1]
```diff
@@ -263,15 +263,15 @@ std::pair<int, int> Network::load_network(std::istream& wtfile) {
     while (std::getline(wtfile, line)) {
         std::vector<float> weights;
```
Comment (glinscott) [inline src/Network.cpp]: Might be a win to reserve more space for weights too.

Comment (Akababa) [reply src/Network.cpp]: image

My profile shows that it doesn't do too much for me

**Comment by Akababa** [comment]
https://www.diffchecker.com/M2sEHr79

It's slightly different for some reason, although I get different results every time I run diff. Is there a deterministic flag?

**Comment by killerducky** [comment]
Use `-t1 -s1`.

**Comment by Akababa** [comment]
Thanks @killerducky https://www.diffchecker.com/9IZB2UWE

**Comment by glinscott** [comment]
Awesome, thanks!

**Pull request merged**